src/components/useConversation.tsx

```diff
@@ -0,0 +1,104 @@
+import { useMutation } from '@tanstack/react-query'
```

Comment:
This ain't really a component, it's a util + hook around conversations. This is the part where the DDD (Domain Driven Design) comes into place. I won't suggest new folders and over-complicate the structure. A simple rename `conversation.helpers.tsx` would be enough for a small app like yours.